### ISSUE TITLE
checki_fping [sf#1122262]

### ISSUE THREAD

**Issue by monitoring-user** [monitoring-plugins/monitoring-plugins#458]
Submitted by i_m_chance on 2005-02-14 11:11:02

Hi,

When i run check_fping <host1> <host2> <host3>
The output is it only checks  <host1>

Here is a sample output

[root@apserv libexec]# ./check_fping -v -H
133.88.30.173 133.88.6.4
/usr/local/sbin/fping -b 56 -c 1 133.88.30.173
133.88.30.173 : [0], 84 bytes, 0.19 ms (0.19 avg, 0% loss)

133.88.30.173 : xmt/rcv/%loss = 1/1/0%, min/avg/max =
0.19/0.19/0.19
FPING OK - 133.88.30.173 (loss=0%, rta=0.190000
ms)|loss=0%;;;0;100 rta=0.000190s;;;0.000000

Your answer is very much appreciated.

Regards
Chancellor Sy
